plugins/catalog-import/src/util/useGithubRepos.ts

```diff
@@ -68,8 +84,35 @@ export function useGithubRepos() {
     return submitPRResponse;
   };
 
+  const checkForExistingCatalogInfo = async (
+    location: string,
+  ): Promise<{ exists: boolean; url?: string }> => {
+    let githubConfig: {
+      repoName: string;
+      ownerName: string;
+      githubIntegrationConfig: GitHubIntegrationConfig;
+    };
+    try {
+      githubConfig = getGithubIntegrationConfig(location);
+    } catch (e) {
+      return Promise.resolve({ exists: false });
+    }
+    return await api
+      .checkForExistingCatalogInfo({
+        owner: githubConfig.ownerName,
+        repo: githubConfig.repoName,
+        githubIntegrationConfig: githubConfig.githubIntegrationConfig,
+      })
+      .catch(e => {
+        throw new Error(
+          `Failed to inspect repository for existing catalog-info.yaml:\n${e.message}`,
```

Comment:
Let's avoid newline characters in error messages. Just use a comma to concatenate.

Reply:
:+1: Modified this and other errors in the file to not have new line.

Reply:
We typically use comma though, and let the entire error object be stringified, so you end up with

```
Error: X is broken, NotAllowedError: Could not Y, NotFoundError: Could not find Z
```

Rather than

```
Error: X is broken: Could not Y: Could not find Z
```

And then just avoid using comma in the actual error messages as much as possible,

Reply:
I find it unreasonably amusing that your message ended with a dangling comma

Reply:
Aight, that makes sense. I modified the lines in that file to conform to conventions.